core/payment/src/processor.rs

```diff
@@ -620,10 +625,16 @@ impl PaymentProcessor {
                 if !allocations.is_empty() {
                     for allocation in allocations {
                         if force {
-                            dao.forced_release_allocation(allocation.allocation_id, None)
-                                .await
+                            release_allocation_after(
+                                db.clone(),
+                                allocation.allocation_id,
+                                None,
+                                None,
+                            )
+                            .await
```

Comment:
Problem with this solution is, that after this function returns, we can still have allocations that we wanted to force release. But it could remain like this for now